beers/models.py

```diff
@@ -80,3 +80,99 @@ class Manufacturer(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class Beer(models.Model):
+    name = models.CharField(max_length=25, db_index=True)
+    style = models.ForeignKey(
+        BeerStyle, models.DO_NOTHING, related_name='beers',
+        # TODO: prevent this being null?
+        blank=True, null=True,
+    )
+    manufacturer = models.ForeignKey(
+        Manufacturer, models.CASCADE, related_name='beers',
+    )
+    in_production = models.BooleanField(default=True)
+    abv = models.DecimalField(
+        'Alcohol content (% by volume)',
+        max_digits=4, decimal_places=2, blank=True, null=True,
```

Comment:
This doesn't match `BeerStyle` ABV which is `max_digits=3, decimal_places=1`

Reply:
Monkeynaut is 7.25% in untappd data (even though modern measurement techniques are only good to about +/- 0.2%). Let's propagate the extra decimal place if we have it.